website/docs/prysm-usage/parameters.md

```diff
@@ -90,20 +92,38 @@ These flags are specific to launching the beacon node.
 ### Base flags
 | Flag        | Usage           |
 | ------------- |:-------------|
-| `--no-custom-config` | Run the beacon chain with the real Phase 0 parameters.
 | `--http-web3provider` | Define an eth1 web3 provider string http endpoint. See [here](/docs/prysm-usage/setup-eth1)
 | `--deposit-contract` |Define a deposit contract address. Beacon chain node will listen logs coming from the deposit contract to determine when validator is eligible to participate.
 | `--rpc-host` | Define an address of the host on which the RPC server should listen. Default: 0.0.0.0
 | `--rpc-port` | Define a RPC port to be exposed by the beacon node. >Value: 4000
-| `--rpc-max-page-size` | Define the max number of items returned per page in RPC responses for paginated endpoints. Default: 500
 | `--tls-cert` |Certificate for secure gRPC. Pass this and the tls-key flag in order to use gRPC securely.
 | `--tls-key` | Key for secure gRPC. Pass this and the tls-cert flag in order to use gRPC securely.
 | `--grpc-gateway-host` | Enable gRPC gateway for JSON requests. Default: 127.0.0.1
 | `--grpc-gateway-port` | Enable gRPC gateway for JSON requests. Default: 3500
-| `--min-sync-peers` | The required number of valid peers to connect with before syncing."
 | `--contract-deployment-block` | Define the ETH1 block in which the deposit contract was deployed. Default: 1960177
+| `--gc-percent` | The percentage of freshly allocated data to live data on which the gc will be run again. Default: 100
+| `--head-sync` | Starts the beacon node with the previously saved head state instead of finalized state.
+| `--slots-per-archive-point` | The slot durations of when an archived state gets saved in the DB. Default: 2048
+| `--weak-subjectivity-checkpoint` | Input in `block_root:epoch_number` format to sync from the weak subjectivity checkpoint.
+| `--disable-sync` | Starts the beacon node without entering initial sync and instead exits to regular sync immediately. Default: 10
+| `--historical-slasher-node` | Enables required flags for serving historical data to a slasher client. Results in additional storage and disk usage.
+| `--slasher-tls-cert` | Certificate for secure slasher gRPC connection. Pass this in order to use slasher gRPC securely.
+| `--slasher-provider` | Define a slasher provider string endpoint. Can either be an gRPC server endpoint. Default: 127.0.0.1:5000
```

Comment:
Default is not set, right?

Reply:
It's by default set, but maybe we just handle it differently.